code/_onclick/ai.dm

```diff
@@ -32,9 +32,9 @@
 
 
 /mob/living/silicon/ai/ClickOn(atom/A, location, params)
-	if(world.time <= next_click)
+	if(world.time <= last_click)
 		return
-	next_click = world.time + 1
+	last_click = world.time + 1
```

Comment:
Setting a var called 'last click' to a point in the future is incredibly weird.

Reply:
This is effectively storing the time before you can click again.
Thus, to pass this check, world.time has to be > than last_click.

What do you think would be a more appropriate name if not this?

Reply:
next_click is more appropiate , it stores the time at which the next click can happen

Reply:
Sure that makes sense now.